src/Symfony/Component/PropertyInfo/composer.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "symfony/property-info",
+    "type": "library",
+    "description": "Symfony Property Info Component",
+    "keywords": [
+        "property",
+        "type",
+        "PHPDoc",
+        "symfony",
+        "validator",
+        "doctrine"
+    ],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Kévin Dunglas",
+            "email": "dunglas@gmail.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=5.3.9"
+    },
+    "require-dev": {
+        "symfony/phpunit-bridge": "~2.7",
+        "symfony/serializer": "~2.7",
+        "phpdocumentor/reflection": "~1.0",
+        "doctrine/annotations": "~1.0"
+    },
+    "conflict": {
+        "phpdocumentor/reflection": "<1.0.7"
```

Comment:
Because it's a `require-dev`, so if you require this package in non-dev (`require` section) with `--prefer-lowest --no-dev` it will install `1.0.0` instead of `1.0.7`.